variables.tf

```diff
@@ -2,8 +2,60 @@
 # Input Variables
 ########################################################################################################################
 
-#variable "my_variable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = "default_value"
-#}
+variable "watsonx_data_name" {
+  type        = string
+  description = "The name of the watsonx.data instance."
+  default     = null
+}
+
+variable "resource_group_id" {
+  type        = string
+  description = "The resource group ID where the watsonx data instance is created."
```

Comment:
`resource_group_id` is not required when passing existing instance, so it needs to have a default value of null. And you should add variable validation for this using the cross variable validation feature that came with tf 1.9